1-JavaScript/Week4/homework/ex3-lemonAllergy.js

```diff
@@ -23,8 +23,12 @@ const fruitBasket = [
   'lemon',
 ];
 
-function sanitizeFruitBasket(/* TODO parameter(s) go here */) {
+function sanitizeFruitBasket(fruitBasket, 'lemon') {
```

Comment:
When you are _**defining**_ a function, you can't create an argument that is a string. Argument names must always be without `'` around it. Does that make sense?